runtime/oti/j9accessbarrier.h

```diff
@@ -145,18 +155,39 @@ typedef struct J9IndexableObject* mm_j9array_t;
 		: (&((elemType*)((((J9IndexableObjectWithDataAddressContiguousFull *)(array))->dataAddr)))[index]))
 
 #define J9JAVAARRAYCONTIGUOUS_EA(vmThread, array, index, elemType) \
-	(((vmThread)->isIndexableDataAddrPresent) \
-		? (((vmThread)->isVirtualLargeObjectHeapEnabled) \
-				? J9JAVAARRAYCONTIGUOUS_WITH_DATAADDRESS_VIRTUALLARGEOBJECTHEAPENABLED_EA(vmThread, array, index, elemType) \
-				: J9JAVAARRAYCONTIGUOUS_WITH_DATAADDRESS_VIRTUALLARGEOBJECTHEAPDISABLED_EA(vmThread, array, index, elemType)) \
-		: J9JAVAARRAYCONTIGUOUS_BASE_EA(vmThread, array, index, elemType))
```

Comment:
I think we are removing only usage of four macros above `J9JAVAARRAYCONTIGUOUS_WITH_DATAADDRESS_VIRTUALL*` and all of them can be removed as well.